.github/workflows/smoketest.yml

```diff
@@ -0,0 +1,161 @@
+name: Smoketest
```

Comment:
Any reasons for the name Smoketest?

Reply:
I use this on other projects, and I just think it's a mildly amusing but accurate name: we don't want the project to go up in smoke!

Reply:
xD that's actually pretty good!